components/reddit/reddit.app.js

```diff
@@ -252,3 +252,17 @@ module.exports = {
     },
   },
 };
+  	  async getNewSubredditLinks(after_link, subreddit){
+ 
+        const newSubredditLinks = await this._makeRequest({
+            path: `/r/${subreddit}/new`,
+		    params: {
+		    	after:after_link
+		    },
+          });
+         
+  		return newSubredditLinks;
+
+        },
+    },
+  };
```

Comment:
This is legacy code from the rebase, right?

Reply:
Right